src/Gemstone.Numeric/Gemstone.Numeric.csproj

```diff
@@ -61,6 +61,10 @@
     <PackageReference Include="Gemstone.Common" Version="1.0.128" Condition="'$(Configuration)'!='Development'" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Geo\" />
+  </ItemGroup>
```

Comment:
Is this necessary? I don't see any of the other subfolders referenced directly like this.

Reply:
Huh - no

Reply:
Removed